metrics/report/example.toml

```diff
@@ -0,0 +1,23 @@
+# [mail] section specifies information for
+# a SMTP server authentication as well as the
+# sender and receipts of the report email.
+# This is in order to send the obtained results
+# from checktool to an email whitelist.
+[mail]
+smtp = "myserver.smtp.com"
+port = "25"
+user = ""
+identity = ""
+pass = ""
```

Comment:
*If* we decide a plain text password is acceptable, we should document clearly the risk of using it (add a warning in the example config file and in the `README`) and ensure the tool refuses to run unless the config file is only readable by the user running the tool.

A couple of other points:

- I'd also rename the `pass` config option to `password` to avoid any possible confusion.
- Can we give this tool a more meaningful name than `report`?